power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/deserializer.hpp

```diff
@@ -27,6 +28,26 @@ namespace power_grid_model::meta_data {
 constexpr auto const& as_array(msgpack::object const& obj) { return obj.via.array; }
 constexpr auto const& as_map(msgpack::object const& obj) { return obj.via.map; }
 // NOLINTEND(cppcoreguidelines-pro-type-union-access)
+
+constexpr void unpack(msgpack::object const& obj, std::floating_point auto& attr) {
+    using namespace msgpack;
+    using namespace std::string_view_literals;
+
+    auto infinity = std::numeric_limits<double>::infinity();
+    if (obj.type == type::STR) {
+        auto const& obj_string = obj.as<std::string_view>();
+        if (obj_string == "inf" || obj_string == "+inf") {
+            attr = infinity;
+        } else if (obj_string == "-inf") {
```

Comment:
```suggestion
        if (obj_string == "inf"sv || obj_string == "+inf"sv) {
            attr = infinity;
        } else if (obj_string == "-inf"sv) {
```

or remove `using namespace std::string_view_literals;`. there's no point to say you're `using` it if you don't use it 😬